INSTALL.Unix.md: error when following instructions for Debian

I followed the instructions for Debian, but when running `make release` I got an error. This is a fresh debian created on Digital Ocean.

## Expected Behavior

Output

```
... done
You can now copy the rel/couchdb directory anywhere on your system.
Start CouchDB with ./bin/couchdb from within that directory.
```

## Current Behavior

Output

```
==> rel (compile)
==> couchdb (compile)
Building Fauxton
/bin/sh: 1: npm: not found
Makefile:368: recipe for target 'share/www' failed
make: *** [share/www] Error 127
```

## Possible Solution

`node` was not installed. I installed the current Node LTS version (v6) following the instructions at https://github.com/nodesource/distributions#debinstall

## Steps to Reproduce (for bugs)

Follow debian install instructions at [INSTALL.LINUX.md (version from July 12)](https://github.com/apache/couchdb/blob/348accf5e8c1e0cdcae4803efc3bad2c7149d362/INSTALL.Unix.md)

## Your Environment

```
  Operating System: Ubuntu 16.04.2 LTS
            Kernel: Linux 4.4.0-81-generic
      Architecture: x86-64
```

I got another issue after installing node LTS and running `make release`

```
make[1]: Entering directory '/root/couchdb/src/docs'
*** Make sure that sphinx-build is installed and on your path
Makefile:63: recipe for target 'sphinx-build' failed
make[1]: *** [sphinx-build] Error 1
make[1]: Leaving directory '/root/couchdb/src/docs'
Makefile:362: recipe for target 'src/docs/build' failed
make: *** [src/docs/build] Error 2
```

I run `sudo apt-get install python-sphinx`. After that `make release` behaved as expected

--- a/INSTALL.Unix.md
+++ b/INSTALL.Unix.md
@@ -44,8 +44,14 @@ You should have the following installed:
  * GNU Compiler Collection      (http://gcc.gnu.org/)
  * libcurl                      (http://curl.haxx.se/libcurl/)
  * help2man                     (http://www.gnu.org/s/help2man/)
- * Python (>=2.7) for docs      (http://python.org/)
- * Python Sphinx (>=1.1.3)      (http://pypi.python.org/pypi/Sphinx)
+ * Python (>=2.7)               (http://python.org/)
+
+To build Fauxton, you should have the following installed:
+ * Node.JS (>=6.x)              (https://nodejs.org/)
+   -- obtainable from NodeSource (https://github.com/nodesource/distributions)
+
+To build the documentation, you should have the following installed:
+ * Python Sphinx (>=1.5)        (http://pypi.python.org/pypi/Sphinx)
 
 It is recommended that you install Erlang OTP R16B03-1 or above where
 possible.  You will only need libcurl if you plan to run the
@@ -54,10 +60,6 @@ installing the CouchDB man pages.  Python and Sphinx are only required
 for building the online documentation. Documentation build can be disabled
 by adding the `--disable-docs` flag to the `configure` script.
 
-For up to date instructions, please see:
-
-  https://cwiki.apache.org/confluence/display/COUCHDB/Installing+CouchDB
-
 ### Debian-based Systems
 
 You can install the dependencies by running:
@@ -65,6 +67,13 @@ You can install the dependencies by running:
     sudo apt-get --no-install-recommends -y install \
         build-essential pkg-config erlang erlang-reltool \
         libicu-dev libmozjs185-dev libcurl4-openssl-dev
+
+You can install the Node.JS dependencies via [NodeSource](https://github.com/nodesource/distributions#installation-instructions).
+
+You can install the documentation dependencies by running:
+
+    sudo apt-get --no-install-recommends -y install \
+        python-sphinx
 
 Be sure to update the version numbers to match your system's available
 packages.
@@ -78,6 +87,14 @@ You can install the dependencies by running:
         erlang-os_mon erlang-xmerl help2man \
         js-devel-1.8.5 libicu-devel libtool perl-Test-Harness
 
+You can install the Node.JS dependencies via [NodeSource](https://github.com/nodesource/distributions#rpminstall).
+
+The built-in packages for Sphinx in RHEL repositories are too old
+to run the documentation build process. Instead, use pip:
+
+    sudo yum install python-pip
+    sudo pip install sphinx
+
 ### Mac OS X
 
 To build CouchDB from source on Mac OS X, you will need to install
@@ -89,6 +106,14 @@ You can then install the other dependencies by running:
 
     brew install autoconf autoconf-archive automake libtool \
         erlang icu4c spidermonkey curl pkg-config
+
+You can install the Node.JS dependencies via the
+[official Macintosh installer](https://nodejs.org/en/download/).
+
+You can install the documentation dependencies by running:
+
+    sudo easy_install pip
+    sudo pip install sphinx
 
 You will need Homebrew installed to use the `brew` command.
 

